logos/serah_kassim/serah_kassim.svg

```diff
@@ -0,0 +1,369 @@
+<svg enable-background="new 0 0 1758 420" height="420" viewBox="0 0 1758 420" width="1758" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><image height="420" width="1758" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAABt4AAAGkCAQAAAAcFZlsAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
```

Comment:
Hi @ichtrojan, this is a base 64 encoded PNG wrapped as an SVG. Kindly replace with a high-quality vector version of the logo.

Reply:
It's done

Reply:
Hi @ichtrojan thanks for you're PR. [Same feedback](https://github.com/PaystackHQ/nigerialogos/pull/170#issuecomment-819511016) applies here